src/p11mkcert.c

```diff
@@ -59,6 +59,9 @@ void print_usage(char *progname)
 	     "  -u <days> : validity (in days, default is 365)\n"
 	     "  -j : import certificate to PKCS#11 token\n"
 	     "  -o <file> : output file for PKCS#10 request (stdout if not specified)\n"
+	     "  -a <algo> : signature algorithm for RSA (default is RSA PKCS#1 v1.5)\n"
```

Comment:
I see other notes about default behavior being pkcs1. Worth adding here in the usage?